server/utils/searchEformValidation.ts

```diff
@@ -30,6 +31,10 @@ const schema = Joi.object({
       'date.format': 'Submission date from must be a valid date',
     })
     .custom((value, helpers) => {
+      // Skip validation if ENVIRONMENT_NAME is 'archive'
+      if (config.environmentName === 'archive') {
```

Comment:
Having the if statement here, means that the missing end date validation in the archive environment is skipped.